### DISCUSSION TITLE
Do not know how to use

### DISCUSSION THREAD

**Discussion by Neo-J1e** [Tyrrrz/YoutubeDownloader#285] [Q&A] [answered]
I am sorry for my stupid question first, but I'm so confused.
I just download the .zip，unzip that file， what to do next?

**Comment by Tyrrrz** [comment, marked as answer]
Download from here: https://github.com/Tyrrrz/YoutubeDownloader/releases

Then run the .exe file

**Reply by Neo-J1e** [reply]
Thanks! 
The browser blocked the first file, so I just download the wrong file(source code). I have retried, and it works!

**Comment by ScottsplaysSus** [comment]
Do you see an .exe file in your directory?(the folder that you extracted the .zip file) If yes, simply run the .exe file by double-clicking or right-click and select open. If no, try redownloading the .zip file and extract the contents again. Might also be an antivirus issue(false positive) so check your antivirus program too. If the antivirus really blocked or deleted the .exe file, exclude the folder for scanning from the antivirus program.

**Reply by Neo-J1e** [reply]
Thank you for your patience！
I now find the .EXE file, it's my negligence(awkward...), sorry for wasting your time, and thank you for your patience again!

**Reply by ScottsplaysSus** [reply]
No problem, have fun using the app :)
